modules/compliance-evaluate-kubeletconfig-rules.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+:_content-type: CONCEPT
+[id="compliance-evaluate-kubeletconfig-rules"]
+= Evaluating `KubeletConfig` rules against default configuration values
+
+{product-title} infrastructure may contain incomplete configuration files at run time, and nodes will assume default configuration values for missing configuration options. Some configuration options can be passed as command line arguments. As a result, the Compliance Operator cannot verify if the configuration file on the node is complete, as it may be missing options used in the rule checks.
+
+To prevent false negative results where the default configuration value passes a check, the Compliance Operator uses the `Node/Proxy` API to fetch the configuration for each node in a node pool, then all configuration options that are consistent across nodes in the node pool are stored in a file that represents the configuration for all nodes within that node pool. This increases the accuracy of the scan results. The results are then stored in `/kubeletconfig/role/<role>`.
```

Comment:
> Generally, when you specify the name of a command, parameter, or option in text, specify the word command, parameter, or option after the name.

Command, parameter, and option names are not translated; without the identifying noun, users might be confused about what the name represents.